azerothcore/azerothcore-wotlk
# fix(DB/creature): Adds spawn and movement to Lady Sathrah

## Changes Proposed:
- adds random movement to Lady Sathrah
- adds another possible spawn point to Lady Sathrah
- links both spawn points together in a pool, so that only one can spawn at a time

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8113

## SOURCE:

see issue

## Tests Performed:

- ran SQL, no errors
- checked in-game


## How to Test the Changes:


1. `.go c 12596` or `.go c 12605`
2. kill and check respawns in only one location
3. also observe movement

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
